hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -73,7 +73,11 @@ open class HmppsQueueService(
     val messagesToReturnCount = min(messageCount, maxMessages)
 
     repeat(messagesToReturnCount) {
-      sqsDlqClient.receiveMessage(ReceiveMessageRequest(dlqUrl).withMaxNumberOfMessages(1)).messages.firstOrNull()
+      sqsDlqClient.receiveMessage(
+        ReceiveMessageRequest(dlqUrl)
+          .withMaxNumberOfMessages(1)
+          .withVisibilityTimeout(1),
```

Comment:
Why 1 and not zero?

Reply:
Good Q .. if its set to zero the repeat loop keeps getting the same message!